<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arm</class>
 <widget class="QWidget" name="Arm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="Person_Label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="SampleLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="rowCount">
            <number>4</number>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <row/>
           <row/>
           <row/>
           <row/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <item row="0" column="1">
            <property name="text">
             <string>+</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string>+</string>
            </property>
           </item>
           <item row="0" column="4">
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>X</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>Y</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>Z</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="Open_Existing">
         <property name="text">
          <string>Open Existing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Select_Person">
         <property name="text">
          <string>Select Person</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Motion Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Motion_Type">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right elbow flex</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left elbow flex</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="New_With_Person">
         <property name="text">
          <string>New with this person</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="New_Record">
         <property name="text">
          <string>New record</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Connect_Button">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Reset_Button">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Capture_Button">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Done_Button">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Clear_Button">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
